pkg/core/pipeline/main.go

```diff
@@ -58,9 +58,9 @@ func (p *Pipeline) Init(config *config.Config, options Options) error {
 	p.Actions = make(map[string]action.Action, len(config.Spec.Actions))
 
 	// Init context resource size
-	p.Report.Sources = make(map[string]*result.Source, len(config.Spec.Sources))
-	p.Report.Conditions = make(map[string]*result.Condition, len(config.Spec.Conditions))
-	p.Report.Targets = make(map[string]*result.Target, len(config.Spec.Targets))
+	p.Report.Sources = make(map[string]result.Source, len(config.Spec.Sources))
+	p.Report.Conditions = make(map[string]result.Condition, len(config.Spec.Conditions))
+	p.Report.Targets = make(map[string]result.Target, len(config.Spec.Targets))
```

Comment:
I'm unsure about this change. As far as I remember, we used references so we could change the objects.
Now that it's copied by value, that will change the behavior right?

Was it done for a specific need ?

Reply:
I forgot the initial intend, and it bother me on Udash side but I'll revert this change. Thanks for the reminder

Reply:
I reverted this change to use  apointer. Thanks for this reminder